product_tax_multicompany_default/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Product Tax Multi Company Default
+=================================
+
+This modules set default taxes when product create and adds a button to set not
+default taxes matching by code.
+
```

Comment:
Add `Configuration` section:

#. As obvious, you need to have several companies.
#. Go to *Accounting > Configuration > Settings*
#. Select the proper company (if you are admin or a user with several companies access), or keep the current one (for a regular user).
#. On *Invoicing & Payments* section, set fields "Default Sale Tax" and "Default Purchase Tax".
```